### PR TITLE
fix: scheduled emails layout

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #4534 

### Description:
Screenshot 2023-02-28 at 17 39 45

### CODE REVIEW

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html [hunk 1]
```diff
@@ -1,490 +1,528 @@
...
+      body {
+          background-color: #e5e8eb;
+          font-family: 'Roboto', sans-serif;
+          -webkit-font-smoothing: antialiased;
+          font-size: 14px;
+          line-height: 1.4;
+          margin: 0;
+          padding: 0;
+          -ms-text-size-adjust: 100%;
+          -webkit-text-size-adjust: 100%;
```
Comment (IrakliJani) [inline packages/backend/src/clients/EmailClient/templates/imageNotification.html:33]: no idea

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html [hunk 2]
```diff
@@ -1,490 +1,528 @@
...
+      }
+
+      table {
+          border-collapse: separate;
+          mso-table-lspace: 0pt;
+          mso-table-rspace: 0pt;
```
Comment (IrakliJani) [inline packages/backend/src/clients/EmailClient/templates/imageNotification.html:39]: 🤔

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html [hunk 3]
```diff
@@ -1,490 +1,528 @@
...
+          width: 100%;
+      }
+
+      table td {
+          font-family: 'Roboto', sans-serif;
```
Comment (IrakliJani) [inline packages/backend/src/clients/EmailClient/templates/imageNotification.html:44]: Are we set on Roboto ? @PriPatel

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html [hunk 4]
```diff
@@ -1,490 +1,528 @@
...
+          margin: 0;
+      }
+
+      .content-block {
+          padding-bottom: 10px;
+          padding-top: 10px;
+      }
+
+      .footer {
+          clear: both;
```
Comment (IrakliJani) [inline packages/backend/src/clients/EmailClient/templates/imageNotification.html:132]: this is a bad practice, why was this necessary?

**Review by IrakliJani — COMMENTED** [review]
if it looks good and is responsive we should not care about CSS best practices here.

Logo should be 2x or even 3x

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html (hunk 3 above)
Comment (PriPatel) [reply packages/backend/src/clients/EmailClient/templates/imageNotification.html:44]: @IrakliJani @yasmin-e Yeah, all our emails (sign up, forgot password etc) use Roboto. Not great, but consistent at least.

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html (hunk 4 above)
Comment (yasmin-e) [reply packages/backend/src/clients/EmailClient/templates/imageNotification.html:132]: don't know, was already there! I can remove it

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html (hunk 2 above)
Comment (yasmin-e) [reply packages/backend/src/clients/EmailClient/templates/imageNotification.html:39]: was already there 😅

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html (hunk 1 above)
Comment (yasmin-e) [reply packages/backend/src/clients/EmailClient/templates/imageNotification.html:33]: same 😅

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html [hunk 5]
```diff
@@ -1,490 +1,528 @@
...
+  <style>
+      /* -------------------------------------
+      GLOBAL RESETS
+  ------------------------------------- */
+
+      /*All the styling goes here*/
+
+      img {
+          border: none;
+          -ms-interpolation-mode: bicubic;
```
[the remark this answers is not in the document]
Comment (yasmin-e) [reply packages/backend/src/clients/EmailClient/templates/imageNotification.html:20]: same 😅

File: packages/backend/src/clients/EmailClient/templates/imageNotification.html [hunk 6]
```diff
@@ -396,40 +355,68 @@
...
                                                         role="presentation"
                                                         border="0"
                                                         cellpadding="0"
                                                         cellspacing="0"
                                                     >
+                                                        <tr>
+                                                            <td
+                                                                class="content-block"
+                                                            >
+                                                                <a
```
Comment (ZeRego) [inline packages/backend/src/clients/EmailClient/templates/imageNotification.html:413]: Are we missing a href in this link?

Comment (yasmin-e) [reply packages/backend/src/clients/EmailClient/templates/imageNotification.html:413]: yes, don't know what it should be pointing to
